### DISCUSSION TITLE
SSL 2.0 and SSL 3.0 supported in mdTlsNative?

### DISCUSSION THREAD

**Discussion by KalleAnka** [wqweto/VbAsyncSocket#38] [General]
Hi,

I see some responses referring to SSL3 in pvTlsGetAlgName.
Is there support for SSL(2/3) via mdTlsNative?
Or is it built for TLS 1.0 - TLS 1.3(if the OS supports them)?

**Comment by wqweto** [comment]
`mdTlsNative` backend supports whatever Schannel on current OS supports (incl. SSL2/3) which depends on which protocols/ciphers are enabled in registry. For instance [this application](https://www.nartac.com/Products/IISCrypto/) is a GUI for tweaking Schannel registry settings.

`pvTlsGetAlgName` dumps various [ALG_ID](https://learn.microsoft.com/en-us/windows/win32/seccrypto/alg-id) constants as declared in `wincrypt.h` and `schannel.h` -- it's used for debugging purposes only.
